README.md

```diff
@@ -6,118 +6,131 @@
 [![Griptape Discord](https://dcbadge.vercel.app/api/server/gnWRz88eym?compact=true&style=flat)](https://discord.gg/gnWRz88eym)
 
 The Griptape CLI is a command-line interface for interacting with features of [Griptape Cloud](https://www.griptape.ai/cloud).
-Today, it provides an emulator for Griptape Cloud Managed Structures, which allows you to run and test your Managed Structures locally. 
+Today, it provides an emulator for Griptape Cloud Managed Structures, which allows you to run and test your Managed Structures locally.
 
 ## Prerequisites
+
 - A GitHub account.
 - Python 3.11.
 - [Poetry](https://python-poetry.org) for running the example client program.
 - Linux operating system functionality. For Windows users, Griptape recommends the [Windows Subsystem for Linux (WSL)](https://learn.microsoft.com/en-us/linux/install) for this functionality.
 
-## Installation 
+## Installation
 
 1. Install griptape-cli
-    ```bash
-    pipx install griptape-cli
-    ```
+
+   ```bash
+   pipx install griptape-cli
+   ```
 
    You can also install a pre-release version from GitHub.
-    ```bash
-    git clone https://github.com/griptape-ai/griptape-cli.git
-    ```
 
-    ```bash
-    pipx install ./griptape-cli --force
-    ```
+   ```bash
+   git clone https://github.com/griptape-ai/griptape-cli.git
+   ```
+
+   ```bash
+   pipx install ./griptape-cli --force
+   ```
+
 2. Verify installation
-    ```bash
-    gt --help
-    ```
+   ```bash
+   gt --help
+   ```
 
 ## Skatepark Emulator
+
 You can use the CLI to spin up Skatepark, a local emulator for Griptape Cloud Managed Structures. It exposes an API that is identical to the one you would interact with when running your Managed Structure in Griptape Cloud.
 Use Skatepark to develop, test, and validate that your program will operate as expected when deployed as a Griptape Cloud Managed Structure. Skatepark gives you confidence that when you bring your Structure into Griptape Cloud as a Managed Structure, it will continue to operate as expected, at scale.
 
 1. Start by creating a new repository in your own Github account from the [Managed Structure Template](https://github.com/griptape-ai/managed-structure-template).
-    1. Make sure you're logged in to GitHub.
-    2. Go to the [Managed Structure Template repo](https://github.com/griptape-ai/managed-structure-template).
-    3. Select the *Use this template* drop-down.
-    4. Choose *Create a new repository*.
-    5. Provide a name for the new repository and (optionally) a description.
-    6. Press the *Create repository* button.
-    7. You now have a repository in your own GitHub account that is a copy of the Managed Structure Template for you to begin working with.
+   1. Make sure you're logged in to GitHub.
+   2. Go to the [Managed Structure Template repo](https://github.com/griptape-ai/managed-structure-template).
+   3. Select the _Use this template_ drop-down.
```

Comment:
why italics vs. bold?

Reply:
This was a bad auto-formatter. Fixed it back to bold.